exporters/etw/include/opentelemetry/exporters/etw/etw_tracer.h

```diff
@@ -399,9 +473,18 @@ class Tracer : public trace::Tracer
       // since Unix epoch instead of string, but that implies additional tooling
       // is needed to convert it, rendering it NOT human-readable.
       evt[ETW_FIELD_STARTTIME] = utils::formatUtcTimestampMsAsISO8601(startTimeMs);
-
+#ifdef ETW_FIELD_ENDTTIME
+      // ETW has its own precise timestamp at envelope layer for every event.
+      // However, in some scenarios it is easier to deal with ISO8601 strings.
+      // In that case we convert the app-created timestamp and place it into
+      // Payload[$ETW_FIELD_TIME] field. The option configurable at compile-time.
+      evt[ETW_FIELD_ENDTTIME] = utils::formatUtcTimestampMsAsISO8601(endTimeMs);
+#endif
       // Duration of Span in milliseconds
       evt[ETW_FIELD_DURATION] = endTimeMs - startTimeMs;
+      // Presently we assume that all spans are server spans
+      evt[ETW_FIELD_SPAN_KIND] = uint32_t(trace::SpanKind::kServer);
```

Comment:
Cannot we read this from the span instead of using a hard-code kind?

Reply:
Let me get back to you on this.